docs/UserGuide.md

```diff
@@ -54,35 +55,92 @@ not be so obvious!
 1. Type the command in the command box and press Enter to execute it. e.g. typing **`help`** and pressing Enter will open the help window.<br>
    Some example commands you can try:
 
-   * **`c-list`** : Lists all employees.
+   * **`list`** : Lists all contacts.
 
-   * **`c-add`**`n/John Doe p/98765432 e/81234567 t/parttime` : Adds an employee
+   * **`add`**`n/John Doe p/98765432 e/81234567 t/part time` : Adds an employee's contact
```

Comment:
This part I think should keep c-add? as somebody said it is the wrong command as can't work

Reply:
Noted, I must have modified it by accident, will change back!